WORKSPACE

```diff
@@ -35,6 +37,10 @@ load(
 
 install_rules_ros2_pip_deps()
 
+load("@pybind11_bazel//:python_configure.bzl", "python_configure")
+
+python_configure(name = "local_config_python")
```

Comment:
Pass here the interpreter target python_interpreter_target and then the new config is going to use that custom interpreter i.s.o. trying to use the system one. This will fix the CI issues.

Reply:
Ah, makes sense! btw always feel free to push to the branch :)

Reply:
So for WORKSPACE passing the `python_interpreter_target` works great, but for MODULE.bazel https://github.com/pybind/pybind11_bazel/issues/66 is getting in the way. Looks like we need to stick with the solution that's on main now and can't get pybind via Bzlmod for now :(

Reply:
pybind11 is tamed now